search-provider/eks-discovery-feed-provider.c

```diff
@@ -550,7 +556,7 @@ handle_article_card_descriptions (EksDiscoveryFeedDatabaseContentProvider *skele
     g_autoptr(EkncQueryObject) query = g_object_new (EKNC_TYPE_QUERY_OBJECT,
                                                      "tags-match-any", tags_match_any,
                                                      "tags-match-all", tags_match_all,
-                                                     "limit", 5,
+                                                     "limit", 365,
```

Comment:
Now that I think of it, it sure would be nice to do this on the database side. But I think that's going to be a lot of yak shaving. I'm also not exactly sure how we would do that, selecting a random thing from a database isn't usually a supported operation.